components/match2/commons/opponent.lua

```diff
@@ -289,19 +289,30 @@ function Opponent.readOpponentArgs(args)
 		}
 
 	elseif partySize == 1 then
+		local playerTeam = args.team or args.p1team
+		if playerTeam then
+			playerTeam = playerTeam:lower():gsub('_', ' ')
```

Comment:
`gsub` should be moved to Module:TeamTemplate (already present in Module:TeamTemplate/Named)

Reply:
https://liquipedia.net/commons/index.php?title=Module%3ATeamTemplate&type=revision&diff=430691&oldid=410338

Reply:
https://liquipedia.net/commons/index.php?title=Module:TeamTemplate&diff=430704&oldid=430691

the nil check shouldn't be there because `mw.ext.TeamTemplate.raw()` doesn't support nil templates.